junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertLinesMatch.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.jupiter.api;
+
+import static java.lang.String.format;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.platform.commons.util.Preconditions.condition;
+import static org.junit.platform.commons.util.Preconditions.notNull;
+
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+/**
+ * {@code AssertLinesMatch} is a collection of utility methods that support asserting
+ * lines of {@link String} equality or {@link java.util.regex.Pattern}-match in tests.
+ *
+ * @since 5.0
+ */
+class AssertLinesMatch {
+
+	static void assertLinesMatch(List<String> expectedLines, List<String> actualLines) {
+		notNull(expectedLines, "expectedLines must not be null");
+		notNull(actualLines, "actualLines must not be null");
+
+		// trivial case: same list instance
+		if (expectedLines == actualLines) {
+			return;
+		}
+
+		int expectedSize = expectedLines.size();
+		int actualSize = actualLines.size();
+
+		// trivial case: when expecting more then actual lines available, something is wrong
```

Comment:
;)

Looking at how http://thenthan.com describes the difference between the two words, I think that "than" is the correct word to use here, since AFAICT we're talking about a comparison between "expected lines" and "actual lines", rather than specifying that something about "expected lines" happens or should happen before "actual lines".

Unless, have I misunderstood the intention of this comment?